docs/JestObjectAPI.md

```diff
@@ -989,6 +989,16 @@ This function is not available when using legacy fake timers implementation.
 
 :::
 
+### `jest.runToFrame()`
```

Comment:
Should we call it `runToNextFrame`? `runToFrame` sounds to me like it'd take which frame (or a number of frames) as an argument

Reply:
I just mirrored the sinon api without too much thought 😊 

`runToNextFrame()` feels nice and fits with `runOnlyPendingTimers()`.
Another idea: `advanceTimersToNextFrame()` - fits in with `advanceTimersToNextTimer()

Which do you lean towards? (Or something else?)

Reply:
yeah, `advanceTimersToNextFrame` sounds perfect to align with existing functions 👍

Reply:
Cc @SimenB

Reply:
hmm? I believe I answered the question? 😅

Reply:
Sorry, for some reason I didn't see your reply